connectivity/ble_indication_hs3001/README.md

```diff
@@ -0,0 +1,249 @@
+
+
+ **_NOTE:_**  In this example we are illustrating only indication
+
+## HW and SW configuration
+
+This example runs on the BLE "SmartBond device family" devices:
+- DA14531 daughter board + DA145xxDEVKT-P PRO-Motherboard.
+
+The user manuals for the development kits & HS3001 can be found:
+- [Here](https://www.renesas.com/eu/en/document/mat/um-b-114-da14531-development-kit-pro-hardware-user-manual) for the DA145xxDEVKT-P PRO-Motherboard.
+- [Here](https://www.renesas.com/eu/en/document/mat/um-b-117-da14531-getting-started-pro-development-kit-html-chinese) for getting started with Pro Development kit.
+- [Here](https://www.renesas.com/us/en/document/dst/hs300x-datasheet) for HS3001.
+
+
+
+## HS3001 pin configuration:
+
+In `user_periph_setup.h` you can check the following configuration for I2c.It is also possible to modify according to your needs:
+
+```c
+#define I2C_SLAVE_ADDRESS           0x44                   // Set slave device address
+#define I2C_SPEED_MODE              I2C_SPEED_FAST        // Speed mode: 	I2C_SPEED_FAST (400 kbits/s)
+#define I2C_ADDRESS_MODE            I2C_ADDRESSING_7B     // Addressing mode: I2C_ADDRESSING_7B
+#define I2C_ADDRESS_SIZE            I2C_1BYTES_ADDR       // Address width: 	I2C_1BYTE_ADDR 
+
+```
+## Hardware configuration:
+Regarding following figure you can connect the sensor to the development board.
+![Motherboard_Hardware_Configuration_DA14531](assets/HS3001_DA1531.svg)
+
+In the same header file you can see the following configuration for HS3001:
+
+```c
+
+#define HS3001_SCL_PORT            GPIO_PORT_0
+#define HS3001_SCL_PIN	           GPIO_PIN_6
+#define HS3001_SDA_PORT            GPIO_PORT_0
+#define HS3001_SDA_PIN             GPIO_PIN_7
+
+```
+
+## How to run the example
+
+### Setup
+For the initial setup of the project that involves linking the SDK to this SW example, please follow the Readme [here](../../Readme.md).
+
+1. Start Keil using the `ble_app_indication_HS3001.uvprojx` Keil project file.
+
+2. Expand the dialog shown in the red box in the image below.
+
+![Expand_Select_Device](assets/Expand_Select_Device.png)
+
+1. Select your device: DA14531, DA14586 or DA14585.
+		
+![Select_Device](assets/Select_Device.png)
+
+
+
+## Expected Results
+
+1. Open the BLE scanner app and look for "REN_HS3001_BLE_IND".
+
+2. Connect to the device.
+
+3. Subscribe to the indication.
+
+If everything went well, you should be able to receive temperature & humidity data as the value of the custom characteristic whenever there is a change in the values, as shown in the image below.
+
+You can use different software on your phone to connect and receive the data. For example:
+
+
+[SmartBond](https://play.google.com/store/apps/details?id=com.renesas.smartbond&hl=it&gl=US)
+
+or
+
+[LightBlue](https://play.google.com/store/apps/details?id=com.punchthrough.lightblueexplorer&hl=nl)
+
+
+
+
+
+
+# In the next figures you can see the expected result on Lightblue
```

Comment:
I would only show the expected result on the smartbond app since this is our own.

Reply:
I had this discussion before with @svdpuranik. Still, I insist on having Lightblue  expected results for at least this example. Even by short checkout Lightblue here has more information to offer about what is what is the goal and how these processes are working.